package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "routific",
-  "version": "0.0.3",
+  "version": "0.0.5",
```

Comment:
I skipped version 0.0.4 since I know there's another pending PR already assigned that version...